builtins/amp-ad.md

```diff
@@ -38,6 +38,12 @@ resources in AMP. It requires a `type` argument that select what ad network is d
     data-aax_src="302">
 </amp-ad>
 ```
+
+#### Styling
+
+`<amp-ad>` elements may not themselves have or be placed in containers that have CSS `position: fixed` set.
```

Comment:
So, one possible case here is the desktop layout where ads are pushed to the right? Are those bad or will we leave them for later?